modules/configuring-pods-static-ip.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/configuring-secondary-networks.adoc
+
+:_content-type: CONCEPT
+[id="configuring-pods-static-ip_{context}"]
+= Configuring pods with a static IP address
+
+The following example provisions a pod with a static IP address.
+
+[NOTE]
+====
+* You can only specify the IP address for a pod's secondary network attachment for a layer 2 or localnet attachment.
+* Specifying a static IP address for the pod is only possible when the attachment configuration does not feature subnets.
+====
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  annotations:
+    k8s.v1.cni.cncf.io/networks: '[
+      {
+        "name": "l2-network", <1>
+        "mac": "02:03:04:05:06:07", <2>
+        "interface": "myiface1",
+        "ips": [
+          "192.0.2.20/24"] <3>
```

Comment:
would we want to close the array in a separate line ? (not sure how this is done on the rest of the docs)